letsdrill.rb

```diff
@@ -1,13 +1,21 @@
 def get_letter_grade(integer)
- 
-  #Put your code here!
-
+  grades = {
+    "A" => 90..100,
+    "B" => 80..89,
+    "C" => 70..79,
+    "D" => 60..69,
+    "F" => 0..59
+  }
+  
+  grades.each do |k,v|
+    if v === integer
+      return k
```

Comment:
Thanks for the input. Your example using division is more what I was hoping to come up with. I just didn't think of it.